source/base/data_out_base.cc

```diff
@@ -7689,7 +7689,6 @@ DataOutInterface<dim, spacedim>::write_vtu_in_parallel(
 #else
 
   const int myrank = Utilities::MPI::this_mpi_process(comm);
-
```

Comment:
instead of ``size`` below, use ``const int n_ranks = Utilities::MPI::n_mpi_processes(comm);`` here.